doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -475,3 +475,17 @@ docker tag docker.io/busybox:1.28.4 myopenshiftcluster/openshift/busybox:latest
 +
 ** For  {OPENSHIFT} version 3.11, see the "Accessing the Registry" section of the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/developer_guide/index[OpenShift Container Platform 3.11 Developer Guide].
 ** For  {OPENSHIFT} version 4.1, see the "Accessing the Registry" section of https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html-single/registry/index[Configuring registries for OpenShift Container Platform 4.1].
+
+== Offline Maven repository
+
+.The `offliner` tool reports errors when downloading artifacts for an offline Maven repository [https://issues.redhat.com/browse/RHPAM-2234[RHPAM-2234]]
+
+Issue: When you run the `offliner` tool to download artifacts for an offline Maven repository, the tool reports that downloading some artifacts has failed. The error message is `The artifact was not found in any of the provided repositories`.
+
+Workaround: If the failed artifacts relate to `installer`, ignore the error messages. These artifacts are not required for the offline Maven repository.
+
+In other cases, for example, for `maven-compiler-plugin` or `docbook` artifacts, specify one more source repository and run the `offliner` command again:
+
+----
+java -jar offliner-<version>.jar -r \https://maven.repository.redhat.com/ga/ -r \https://repo1.maven.org/maven2/ -r \https://repository.jboss.org/nexus/content/groups/public/ -d /home/user/temp {PRODUCT_FILE}-offliner.txt
+----
```

Comment:
If you put the \ in front of https to prevent the link form formatting as clickable, please remove. That is only required for inline text, not in block code formatting.

Reply:
You need to define <version>. Also is PRODUCT_FILE  a variable? If so, it should be in <> and I guess in the {} as well:
{<PRODUCT_FILE>}. But I am only guessing. Would be good to confirm with SME, tomorrow, but go ahead and merge tonight. Also, I like to put variables in all caps.